app/assets/stylesheets/application.scss

```diff
@@ -123,3 +123,7 @@ h6 {
 .box-height {
   height: 100px
 }
+
+.nav-feature {
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.